homeassistant/components/climate/sensibo.py

```diff
@@ -106,12 +106,13 @@ def async_assume_state(service):
 class SensiboClimate(ClimateDevice):
     """Representation of a Sensibo device."""
 
-    def __init__(self, client, data):
+    def __init__(self, hass, client, data):
```

Comment:
No need for this. `self.hass` is set when the entity is added to Home Assistant.